roles/open_notificaties_docker/defaults/main.yml

```
@@ -50,7 +50,7 @@ opennotificaties_rabbitmq_pass: rabbit
 
 opennotificaties_publish_broker_url: "amqp://{{ opennotificaties_rabbitmq_user }}:{{ opennotificaties_rabbitmq_pass }}@message-broker:5672/%2F"
 opennotificaties_celery_broker_url: "amqp://{{ opennotificaties_rabbitmq_user }}:{{ opennotificaties_rabbitmq_pass }}@message-broker:5672//"
-opennotificaties_celery_result_backend: "{{ opennotificaties_celery_broker_url }}"
+opennotificaties_celery_result_backend: "redis://opennotificaties-cache:6379/1"
```

this DNS name doesn't seem right, the container is deployed with `cache` as name (see https://github.com/open-zaak/ansible-collection/blob/main/roles/open_notificaties_docker/tasks/containers.yml#L7)

I would also like to keep the option to override the DB index, just like how `opennotificaties_cache_db` can be overridden

```suggestion
opennotificaties_celery_result_backend: "redis://cache:6379:{{ opennotificaties_celery_result_db|default(1) }}"
```